src/bridge/EspressoTEEVerifier.sol

```diff
@@ -136,13 +146,6 @@ contract EspressoTEEVerifier is Ownable {
         success = true;
     }
 
-    /*
```

Comment:
we dont need this as we can call `transferOwnership` function present in the Ownable contract